src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--month-length--29 calendar--start-day--sun">
+      <div class="calendar__date calendar__date--1"></div>
```

Comment:
Just use nth-child for this instead use extra class
```suggestion
      <div class="calendar__date"></div>
```